### PR TITLE
Correct license

### PR DESCRIPTION
**Pull request by AndreyLukashenko**

### CODE REVIEW

**Comment by AndreyLukashenko** [comment]
All right 🙀

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "url": "https://github.com/DaQuirm/css-quickdraw-redux.git"
   },
   "author": "",
-  "license": "LGPLv3",
+  "license": "LGPL-3.0",
```
Comment (DaQuirm) [inline package.json]: RMS approves :cat:

**Comment by DaQuirm** [comment]
LGTM

**Comment by kirilknysh** [comment]
Any license is OK except the one that triggers warnings :+1:

**Pull request merged**